### DISCUSSION TITLE
stoicism/time/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#22] [General]
# stoicism/time/

https://shanenull.com/stoicism/time/

**Comment by shane0** [comment]
# define and compute

- is time a particle or a wave?  wave

## maximum entropy

- far as we can tell particles follow some rules
- we can predict the outcome in controlled scenarios
- outside scenarios ~50% of motion is random or unpredictable
- the input for the computation of time is the state of the system which seems infinite
- the algorithms are the laws of physics + quantum physics
- the out put is a new state with the entropy applied
- so the universe is driven by entropy which are like magnets trying to stick together
- the + attracts the - 
- except sometimes the - and - touch so they repel
- but eventually the atoms break down 
- the periodic table elements theirselves will eventually dissolve
- heat death
- it will all cool down until everything is a luke warm soup
- with no temperature difference
- with no volumetric difference
- with no electrical difference
- with no magnetic difference
- with no gravitational difference

but if anyone survives long enough to witness this the ancients believe something starts the entire cycle all over

like what we call the big bang will end in a big nothing

but then it will start all over

but we will have no trace or evidence of the transition because when everything is in balance

which we call chaos ironically

when everything balances out wouldn't that be order?  things are put back in order

then something disturbs that state of pure balance and creates divided states that will again fight to return to balance

this might be why humans act the way they do

we infight because our atoms themselves are infighting to return to maximum entropy

our minds would like to believe this is a process not an event 

otherwise the universe is facing annihilation 

## periodic table

- https://youtu.be/hePb00CqvP0?si=YerWXwxAUI8RDvsM

## robot definition


- Time** is a fundamental concept that measures the progression of events and their duration
- a dimension that allows us to 
	- sequence events 
	- understand their relationship to each other

ways to think about time:

* **A continuum:** Time flows continuously, from the past to the present and into the future.
* **A measurement:** We measure time using units like seconds, minutes, hours, days, years, etc.
* **A concept:** Time is a human construct that helps us understand and organize our experiences.

 we can measure and quantify time
- its true nature and existence remain a subject of philosophical and scientific debate
 

## first principles and time

- collect assumptions: we only have guesses we can measure and quantify time
- question them
- is this a question discussion problem

### unanswerable questions¶

- buddha believed these not conducive to spiritual progress.

The 14 unanswerable questions are:

- which relate to time

Do you exist? time would not care without time we do not exist we would be motionless
Is the world eternal or not? this is related but not knowable 
Is the world finite or infinite? also not knowable
Is the soul the same as the body or different? time would not care unless soul is multi dimensional
Does a person who has attained enlightenment come back after death? time does not care
Are there past and future lives? time does not care
Is there free will or determinism? time would not care
Can the world be known? time would not care
Is the world one or many? time would not care
Is there a God? time would not care
Why is there evil and suffering in the world?  time would not care
When will the Buddha come back to the world? time would not care
When will the world end? time would not care
What is the nature of consciousness? time would not care


## robot responses 

- Time** is a fundamental concept that measures the progression of events and their duration
- a dimension that allows us to 
	- sequence events 
	- understand their relationship to each other

ways to think about time:

* **A continuum:** Time flows continuously, from the past to the present and into the future.
* **A measurement:** We measure time using units like seconds, minutes, hours, days, years, etc.
* **A concept:** Time is a human construct that helps us understand and organize our experiences.

 we can measure and quantify time
- its true nature and existence remain a subject of philosophical and scientific debate
 
**While we can't definitively prove the exact nature of time, we can make certain observations and deductions:**

1. **Time as a measurement:** We can measure time using various methods, from sundials to atomic clocks. This means we can quantify and compare durations.
2. **Time as a dimension:** In Einstein's theory of relativity, time is considered a fourth dimension alongside space. This suggests that time is a fundamental aspect of our universe.
3. **Time's arrow:** We generally experience time as flowing in one direction (from past to future). This is related to the concept of entropy, which suggests that systems tend to become more disordered over time.
4. **Time dilation:** According to relativity, time can pass at different rates depending on factors like speed and gravity. This has been experimentally confirmed.

**Reply by shane0** [reply]
https://youtu.be/-6rWqJhDv7M

**Reply by shane0** [reply]
# dismantle time

- what is "now" concept does not map breaks easily
- the state S is the input S+ is the next
- humans have blindspods and expectations regarding time
- the past looks fixed
- the future is invisible and unpredictable
- we measure time with clocks
- now is not graspable outside bubbles

**Reply by shane0** [reply]
- past != present != future but is the input

**Comment by shane0** [comment]
deep problem

https://youtu.be/PZ_qnBwSejE

**Reply by shane0** [reply]
https://github.com/shane0/stoicism/issues/29

**Comment by shane0** [comment]
# imagine 

- no space
- no objects
- no time

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=-6rWqJhDv7M
